power-pay-backend/src/main/java/com/adorsys/gis/powerpay/powerpaybackend/services/userRegistrationController.java

```diff
@@ -0,0 +1,52 @@
+package com.adorsys.gis.powerpay.powerpaybackend.services;
+
+import com.adorsys.gis.powerpay.powerpaybackend.domain.User;
+import com.adorsys.gis.powerpay.powerpaybackend.domain.UserRegistration;
+import jakarta.validation.Valid;
+import lombok.Getter;
+import lombok.Setter;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api/registration")
+public class userRegistrationController {
```

Comment:
Naming conventions in Java are very important. Change `userRegistrationController` to `UserRegistrationController`